{"playernames": [null, "python.exe", "python.exe", "python.exe"], "error": "Traceback (most recent call last):\n  File \"D:\\Temp\\b2ki\\b2ki\\tools\\engine.py\", line 93, in run_game\n    sandbox.start(bot_cmd)\n  File \"D:\\Temp\\b2ki\\b2ki\\tools\\sandbox.py\", line 385, in start\n    raise SandboxError('Failed to start {0}'.format(shell_command))\nSandboxError: Failed to start ['D:/Temp/b2ki/b2ki/tools/../your', 'bot/bin/Debug/MyBot.exe']\n"}